Remotion/Security/Core/Metadata/MetadataToXmlConverter.cs

```diff
@@ -98,7 +98,8 @@ private void AppendEnumValueInfoAttributes (XmlDocument document, EnumValueInfo
       XmlAttribute enumValueIDAttribute = document.CreateAttribute ("id");
       enumValueIDAttribute.Value = enumValueInfo.ID;
 
-      EnumWrapper enumWrapper = EnumWrapper.Get(enumValueInfo.Name, enumValueInfo.TypeName);
+      // TODO: Add debug notnull assertions
+      EnumWrapper enumWrapper = EnumWrapper.Get(enumValueInfo.Name!, enumValueInfo.TypeName);
```

Comment:
Same rule as with LocalizedName in other file. leave the bang (arg checks exist) and link the follow-up task/todo from before.